.editorconfig

```diff
@@ -1,6 +1,10 @@
 [*.{kt,kts}]
 ij_kotlin_allow_trailing_comma = true
 ij_kotlin_allow_trailing_comma_on_call_site = true
+ij_java_use_single_class_imports = true
+ij_groovy_use_single_class_imports = true
+ij_kotlin_name_count_to_use_star_import = 100
+ij_kotlin_name_count_to_use_star_import_for_members = 100
```

Comment:
These settings keep getting reverted for me; I've added them to the editorconfig in the hopes that it may solve the issue for myself and others.